src/plugin.ts

```diff
@@ -21,7 +21,7 @@ export function buildEnd(err?: Error): void {
 }
 
 function useStyleSheetTransform(filename: string) {
-  return filename.match(/\.s?(a|c)ss/);
+  return path.extname(filename).match(/s?(a|c)ss/);
```

Comment:
Can we change this regex to `/.s?(a|c)ss$/`?

This would match from the end of string and ensure the cases of `.css` / `.scss` / `.sass` are matched only.